core/build.gradle

```diff
@@ -1,8 +1,11 @@
+description = 'A lightweight, high-performance Java instrumentation library'
+
 dependencies {
-    compile group: 'org.slf4j', name: 'slf4j-api', version:'1.7.25'
+    api group: 'org.slf4j', name: 'slf4j-api', version:'1.7.25'
```

Comment:
Hmm, is there need for this to be `api`? I don't remember any slf4j classes being visible in the public API but I might remember wrong...

Reply:
It's needed because of the SLF4JReporter. Otherwise it will give compilation error to the clients unless they add this dependency explicitly.

Reply:
Oh, that's in core. Then it is indeed `api` scoped.